server/controllers/user.js

```diff
@@ -0,0 +1,86 @@
+import models from '../models';
+import jwt from 'jsonwebtoken';
+const user = models.User;
+
+const UserController = {
+  addUser(req, res) {
+    return user 
+        .create({
+            lastName: req.body.lastName,
+            firstName: req.body.firstName,
+            username: req.body.username,
+            password: req.body.password,
+            email: req.body.email,
+            active: req.body.active
+        })
+        .then(() => {
+            res.send ({
+                status: 'Success',
+                message: "User successfully added"
+            });
+        })
+        .catch(error => res.status(400).send({
+            status: false,
+            message: error
+        }));
+  },
```

Comment:
no space to differentiate `create` from `login`.
Also consider including `jwt` when creating users